spec/validators/has_at_least_one_validator_spec.rb

```diff
@@ -35,18 +35,18 @@ class Breakfast < Struct.new(:ingredients)
     it 'has an error' do
       expect(ingredient_errors.length).to eql(1)
       expect(ingredient_errors.first).to \
-        eql('ingredients should have at least one of [:bacon, :eggs, :beans]')
+        eql('should have at least one of bacon, eggs, or beans')
     end
   end
 
-  context 'Our ingredients are not an array' do
+  context 'Our ingredients are not an array (non-human error)' do
     let(:ingredients) { :'A single hash brown' }
 
     it { is_expected.to be_invalid }
     it 'has an error' do
       expect(ingredient_errors.length).to eql(1)
       expect(ingredient_errors.first).to \
-        eql('ingredients should be an enumerable')
+        eql('should be an enumerable')
```

Comment:
One possibility is to take it out completely. When you do this the error is raised when you attempt to save the record – but it's a bit cryptic, it's a `PG::Error` related to trying to stuff a string into an `ARRAY` field.